src/system/data-attributes-product.md

```diff
@@ -63,19 +63,19 @@ The installation used to export this data has the sample data installed, and has
 |use_config_min_qty|Determines if the default value from the configuration is used, and corresponds to the Use Config Settings checkbox. Values:<br/>**1** — (Yes) The default configuration setting is used for the value of this attribute.<br/>**0** (or blank) — (No) The default configuration can be overridden for the value of this attribute.|
 |is_qty_decimal|Determines if the qty attribute has a decimal value. Values:<br/>**1** — (Yes) The value of the qty attribute is a decimal value.<br/>**0** (or blank) — (No) The value of the qty attribute is a whole number (integer).|
 |allow_backorders|Determines if your store allows backorders, and how they are managed.|
-|use_config_backorders|Determines if the default configuration setting for backorders is used, and corresponds to the state of the Use Config Settings checkbox. Values:<br/>**1** — (Yes) The value of the qty attribute is a decimal value.<br/>**0** (or blank) — (No) The value of the qty attribute is a whole number (integer).|
```

Comment:
It seems that the documentation for the `is_qty_decimal` field was somehow copied into many of the other fields.